tests/4.5/target/test_target_private.c

```diff
@@ -35,26 +33,22 @@ int main() {
       compute_array[i][j] = 0;
 
 
-  omp_set_num_threads(4);
-#pragma omp parallel
-{
-  int fp_val = omp_get_thread_num();
-  int p_val=0;
-
-  real_num_threads = omp_get_num_threads();  
+    
 
-#pragma omp target map(tofrom:compute_array) firstprivate(fp_val) private(p_val)
+#pragma omp target map(tofrom:compute_array, real_num_threads) private(p_val)
+{
+#pragma omp parallel private(p_val,i) num_threads(4)
   {
-    p_val = fp_val;
-
+    p_val = omp_get_thread_num();
+    real_num_threads = omp_get_num_threads();
     for (i = 0; i < N; i++)
-      compute_array[p_val][i] = 100;
-  } // end target
-}//end parallel
+      compute_array[p_val][i] += p_val;
+  } // end parallel
+}//end target
 
```

Comment:
Add warning about real_num_threads == 1 (can't properly test private as mentioned in the meeting)
Also maybe add warning about num_threads < 1 (this has been done in some past tests)?